frontend/components/SetupPage/SetupYourAgent/SetupYourAgent.tsx

```diff
@@ -151,6 +153,10 @@ const SetupYourAgentForm = ({ serviceTemplate }: SetupYourAgentFormProps) => {
               ...serviceTemplate.env_variables.TWIKIT_PASSWORD,
               value: values.xPassword,
             },
+            TWIKIT_COOKIES: {
+              ...serviceTemplate.env_variables.TWIKIT_COOKIES,
+              value: cookies,
```

Comment:
this will write it in the format of:
```
'{"guest_id_marketing":"111","guest_id_ads":"111","personalization_id":"111","guest_id":"111","kdt":"111","twid":"111","ct0":"111","auth_token":"111","att":"111"}'
```

Reply:
there's only a key named `_twitter_sess` missing when I compare it to the cookies that are generated by the BE validation. @dvilelaf do you know if that will cause any issues?

Reply:
I think it is needed, but I'm not 100% sure. I definitely have it in my cookies